pkg/archive/archive.go

```diff
@@ -525,7 +525,7 @@ loop:
 		if err != nil {
 			return err
 		}
-		if strings.HasPrefix(rel, "..") {
+		if strings.HasPrefix(rel, "../") {
```

Comment:
@ahmetalpbalkan I'll open a `white-label` issue for making sure paths use filepath :)